### PR TITLE
Assignment submitted

### PR DESCRIPTION
**Pull request by davasuccess**

### CODE REVIEW

File: dailySales.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        <tr class="lead container bg-primary">
+            <th>Select</th>
+            <th>S/No.</th>
+            <th>Drug name</th>
+            <th>Price</th>
+            <th>Payment Status</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr class="text-muted">
```
Comment (jhilton11) [inline dailySales.html]: Please remove the text-muted class. There's no need to make the first table row text-muted.

**Review by jhilton11 — CHANGES_REQUESTED** [review]
Your table data must contain productId, name of item , unit price and quantity sold for the day.
You also did not calculate the total sales for the day.
The first item of the table cannot be deleted. Please fix that.
Overall the web UI is good. That's a good thing you did.

File: dailySales.html (hunk 1 above)
Comment (davasuccess) [reply dailySales.html]: correction has been made excluding the  addition of the total

**Review by jhilton11 — CHANGES_REQUESTED** [review]
The spelling of quantity on the table row is wrong.
The first item of the table cannot be deleted still. Please fix that.
The title of the last column is supposed to be total and not payment status
The total for each column is unit_price * quantity
The total sales is going to be the sum of the total of each row.

File: dailySales.html [hunk 2]
```diff
@@ -19,7 +19,8 @@
     <div class="container">
             <form action="" class="my-5 mx-5">
                 <input type="text" name="item" id="drug-name" placeholder="Drug name">
-                <input type="text" name="price" id="drug-price" placeholder="cost">
+                <input type="text" name="item-id" id="drug-id" placeholder="aci90045">
```
Comment (jhilton11) [inline dailySales.html]: placeholder="drug id"

File: dailySales.html [hunk 3]
```diff
@@ -59,23 +69,49 @@
 $(document).ready(function(){
     $(".add-item").click(function(){
         var name = $("#drug-name").val();
+        var drugId = $('#drug-id').val();
         var price = $("#drug-price").val();
-        var no = 01; 
+        var quantity = "";
```
Comment (jhilton11) [inline dailySales.html]: quantity is not equal = "" !!!!!!
Use JQuery to find the value entered

**Pull request merged**